packages/installation-tests/esm-playwright-firefox.mjs

```diff
@@ -14,11 +14,15 @@
  * limitations under the License.
  */
 
-import { firefox, selectors, devices, errors } from 'playwright-firefox';
+import { chromium, firefox, webkit, selectors, devices, errors } from 'playwright-firefox';
```

Comment:
nit: at this point I'd unify all three scripts into a one that accepts browser to launch as an argument